tests/plugins/test_koji_import.py

```diff
@@ -1855,11 +1787,9 @@ def test_koji_import_set_manifest_list_info(self, caplog, tmpdir, monkeypatch, o
                 assert by_tag == '%s/myproject/hello-world:%s' % (registry,
                                                                   version_release)
 
-    @pytest.mark.skipif(not PULP_SYNC_AVAILABLE,
-                        reason="pulp_sync not available")
     @pytest.mark.parametrize('available,expected', [
         (None, ['sha256:v1', 'sha256:v2']),
-        (['foo', 'sha256:v1'], ['sha256:v1']),
+        (['foo', 'sha256:v1'], ['sha256:v1', 'sha256:v2']),
         (['sha256:v1', 'sha256:v2'], ['sha256:v1', 'sha256:v2']),
     ])
     def test_koji_import_unavailable_manifest_digests(self, tmpdir, os_env,
```

Comment:
I think this whole test should go. We're no longer using `available`, and without that parameter I'm not sure what it's testing.

Reply:
Fixed.